python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 1.4.1
+
+### Feature:
+- Implement `check_table` Operator to validate data quality at table level [#1239](https://github.com/astronomer/astro-sdk/pull/1239)
+- Add `check_column` Operator to validate data quality for columns in a table/dataframe [#1239](https://github.com/astronomer/astro-sdk/pull/1239)
+
+## Bug fixes
+- Support "s3" conn type for S3Location [#1647](https://github.com/astronomer/astro-sdk/pull/1647)
```

Comment:
This is not a bug fix. Is it? This was added to the airflow and we support it too right?

Reply:
This bug, this got removed from airflow but wanted to support for older version